src/App.js

```diff
@@ -0,0 +1,24 @@
+import { useSelector } from 'react-redux';
+import { Route, Routes } from 'react-router-dom';
+import Layout from './components/Layout';
+import Rockets from './routes/Rockets';
+import Missions from './routes/Missions';
+import MyProfile from './routes/MyProfile';
+import NotFound from './routes/NotFound';
+
+function App() {
+  const { rocketList, status, myRockets } = useSelector((store) => store.rockets);
+  return (
+    <Routes>
+      <Route path="/" element={<Layout status={status} />}>
+        <Route index element={<Rockets rockets={rocketList} />} />
+        <Route path="rockets" element={<Rockets rockets={rocketList} />} />
+        <Route path="my-profile" element={<MyProfile myRockets={myRockets} />} />
+        <Route path="missions" element={<Missions />} />
+        <Route path="*" element={<NotFound />} />
+      </Route>
+    </Routes>
```

Comment:
- [ ] kindly fix the root route bug, when I start the server the app opens the `http://localhost:3000/space-traveller` link as a root so it causes a 404 error , kindly fix it 🙏 


![image](https://user-images.githubusercontent.com/89282221/224313943-a5c4cb91-96e8-4ca0-afe4-0eab3fe5668d.png)

![image](https://user-images.githubusercontent.com/89282221/224313801-dfed2953-8c97-463e-bebc-2ed7ab97c582.png)